app/controllers/katello/api/v2/content_exports_controller.rb

```diff
@@ -0,0 +1,62 @@
+      history = history.with_content_view_id(params[:content_view_id]) unless params[:content_view_id].blank?
+      respond_with_template_collection("index", 'content_view_version_export_histories',
+                                       :collection => scoped_search(history, 'id', 'asc', resource_class: ContentViewVersionExportHistory))
+    end
+
+    api :GET, "/content_exports/export_api_status", N_("true if the export api is pulp3 ready and usable. This API is intended for use by hammer-cli only.")
+    def export_api_status
+      ::Foreman::Deprecation.api_deprecation_warning("export_api_status is being deprecated and will be removed in a future version of Katello.")
+      render json: { api_usable: SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE) }, status: :ok
+    end
+
+    api :POST, "/content_exports/export", N_("Export a content view version.  Relevant only for Pulp 3 repositories")
+    param :id, :number, :desc => N_("Content view version identifier"), :required => true
+    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3"), :required => true
+    param :chunk_size_mb, :number, :desc => N_("Chunk export-tarfile into pieces of chunk_size mega bytes."), :required => false
+    param :from_history_id, :number, :desc => N_("Export history identifier used for incremental export."), :required => false
+    def export
+      fail HttpErrors::BadRequest, _("Invalid usage for Pulp 2 repositories. Use export for Yum repositories") unless SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE)
+      invalid_params = [:export_to_iso, :iso_mb_size, :since].reject { |param| params[param].blank? }
+      unless invalid_params.empty?
+        fail HttpErrors::BadRequest, _("Invalid parameters provided - %{params}. These are only relevant for Pulp 2 repositories" % { params: invalid_params.join(', ')})
+      end
+
+      if params[:destination_server].blank?
+        fail HttpErrors::BadRequest, _("Destination Server Name required for Pulp3 repositories")
+      end
+
+      history = ::Katello::ContentViewVersionExportHistory.find(params[:from_history_id]) unless params[:from_history_id].blank?
+
+      tasks = async_task(::Actions::Pulp3::Orchestration::ContentViewVersion::Export, @version, destination_server: params[:destination_server],
+                                                                                         chunk_size: params[:chunk_size_mb],
+                                                                                         from_history: history)
+
+      respond_for_async :resource => tasks
+    end
+
+    private
+
+    def find_exportable_content_view_version
+      @version = ContentViewVersion.exportable.find(params[:content_view_version_id])
+      throw_resource_not_found(name: 'content_view_versions', id: params[:content_view_version_id]) if @version.blank?
```

Comment:
```suggestion
      throw_resource_not_found(name: 'content_view_versions', id: params[:id]) if @version.blank?
```

Reply:
the `name` should be 'content view version' because it gets used in a sentence like 'can not find content view version with id $ID'

Reply:
Fixed